locale/fi/LC_MESSAGES/django.po

```diff
@@ -3572,27 +3573,41 @@ msgstr "Vaativammat pisteet käytetään helpompien vaatimuksien täyttöön."
 msgid "{:d} points"
 msgstr "{:d} pisteet"
 
-#: userprofile/templates/userprofile/login.html:5
-#: userprofile/templates/userprofile/login.html:10
+#: userprofile/templates/userprofile/login.html:6
+#: userprofile/templates/userprofile/login.html:17
 msgid "Login"
 msgstr "Kirjaudu sisään"
 
-#: userprofile/templates/userprofile/login.html:54
-msgid "Login Using Google"
+#: userprofile/templates/userprofile/login.html:61
+msgid "Log in using Google"
 msgstr "Kirjaudu Google-tunnuksella"
 
-#: userprofile/templates/userprofile/login.html:81
+#: userprofile/templates/userprofile/login.html:72
+msgid "Local users"
+msgstr "Paikalliset käyttäjät"
+
+#: userprofile/templates/userprofile/login.html:75
+#, python-format
+msgid ""
+"If you have been provided with credentials specifically for %(brand_name)s, "
+"use this login."
+msgstr ""
+"Mikäli sinulla on tunnukset yksinomaan %(brand_name)s-palvelua varten, "
+"kirjaudu sisään tästä."
+
+#: userprofile/templates/userprofile/login.html:87
 msgid "Password"
 msgstr "Salasana"
 
-#: userprofile/templates/userprofile/login.html:94
+#: userprofile/templates/userprofile/login.html:98
+#: userprofile/templates/userprofile/login.html:99
+msgid "Show more login options"
+msgstr "Näytä lisää kirjautumisvaihtoehtoja"
+
+#: userprofile/templates/userprofile/login.html:107
```

Comment:
DId you makemessages and compilemessages after yesterday's commit? 

For me the translations seem to be a bit off on the container and locally: 
![image](https://user-images.githubusercontent.com/43036333/84795634-6ccbb700-b000-11ea-872a-a33d3f4c51dc.png)

And for instance this line is currently on a different line in the login.html, which would suggest that there has been changes in the translations after the previous message compiling.

Reply:
The `compilemessages` must be run locally, as the binary files are not stored in the git.

Reply:
I didn't update the translations, it seems like only the lines have changed on three of the translations (and the lines actually don't matter, they're just to help the life of the translators by helping to find the context, they are commented out). I could do one more fixup commit with that would update the line numbers. @raphendyr should I?

Reply:
Running A+ in the container should update the translations when you mount it to src (rather than srv).
I talked with Jaakko, he said not to update the translations (since it doesn't actually affect the content), and we talked that it would be good to update the translation-thingy so the comments don't include the line numbers (just the name of the file), since the line numbers can become rather annoying with git.

Reply:
Sounds like a valid idea! The lines are a bit unnecessary in my opinion as well. And good to know that about the behavior of translations, I remembered something about src and srv working differently with translations, and apparently this was it. So this thing can be resolved :)